<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorLasso</class>
 <widget class="QWidget" name="InspectorLasso">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>93</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="accessibleName">
   <string>Lasso Inspector</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="elementName">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Lasso</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="horizontalSpacing">
      <number>3</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Position</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QDoubleSpinBox" name="posY">
       <property name="accessibleName">
        <string>Vertical position</string>
       </property>
       <property name="suffix">
        <string>mm</string>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="maximum">
        <double>999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="sizeHeight">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="accessibleName">
        <string>Height</string>
       </property>
       <property name="suffix">
        <string>mm</string>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="maximum">
        <double>99999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="sizeWidth">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="accessibleName">
        <string>Width</string>
       </property>
       <property name="suffix">
        <string>mm</string>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="maximum">
        <double>99999.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Size</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="posX">
       <property name="accessibleName">
        <string>Horizontal Position</string>
       </property>
       <property name="suffix">
        <string>mm</string>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="maximum">
        <double>999999.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>posX</tabstop>
  <tabstop>posY</tabstop>
  <tabstop>sizeWidth</tabstop>
  <tabstop>sizeHeight</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
